### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by shubhamgoel90**

### CODE REVIEW

File: getting-started/c++/obstacle-avoiding-bot-c++/obstacle-avoiding-bot-c++.md [hunk 1]
```diff
@@ -33,16 +33,16 @@ To program the component
 	cd src
 	gedit specificworker.cpp
 
-Here we will be using both the interfaces we have learnt so far. The entire algorithm is as follows
+Here we will be using both the interfaces we have learned so far. The entire algorithm is as follows
```
Comment (orensbruli) [inline getting-started/c++/obstacle-avoiding-bot-c++/obstacle-avoiding-bot-c++.md]: https://www.grammar.com/learned_vs._learnt

Comment (shubhamgoel90) [reply getting-started/c++/obstacle-avoiding-bot-c++/obstacle-avoiding-bot-c++.md]: @orensbruli Sorry for the inconvenience. I am doing the changes.

**Comment by shubhamgoel90** [comment]
@orensbruli I have made the changes. And again sorry for the inconvenience.

**Comment by orensbruli** [comment]
No inconvenience! This is the common way to get things fixed. Thank you.

**Pull request merged**